db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
```

Comment:
Whats this file about?

Reply:
The seeds.rb file is where the data to initialize the app is stored. Some apps may require initial data to begin with to work as expected. We can use rake task such as the `rake db:setup` to then create the database and initialize with this seed data.